### ISSUE TITLE
Option to add notes on a player.

### ISSUE THREAD

**Issue by mibby** [ruany/LiteBans#24]
Would it be possible to have a way for staff to add notes on a player and allow looking up what is written? Similar to warnings but without it ever expiring or with actions.

**Comment by ruany** [comment]
I've considered adding notes but I'm not completely sure how to implement a notes system in a useful way, and people who need notes probably already have plugins which handle /notes (which would result in a plugin conflict if I added it).

Problems:
- Should the plugin send a notification every single time a player with notes joins the server? If so, this could end up spamming the chat on larger servers
- Are staff members expected to use /notes on a player every time they do something regarding that player? If so, they might forget to do this.

**Comment by mibby** [comment]
> Should the plugin send a notification every single time a player with notes joins the server? 

I don't think so but it can always be a configurable option similar to the notice if someone logs in with a banned account on the same IP.

> Are staff members expected to use /notes on a player every time they do something regarding that player?

My staff would mainly use it for important notes regarding someone being toxic or has a condition they need to abide by. Such as a form of warning and reminder they did x but not severe enough to be banned for.

Looking around, I haven't found any notes plugin that is kept updated and functional. It would be nice to have all punishment type systems inclusive under one plugin, but this is mainly just something some of my staff have been asking for.

**Comment by mibby** [comment]
It would also be nice if you could have a notice when a player logs in if they have notes written down under their name so online staff know that someone who should be watched (with notes) logged in.

**Comment by plasticblues** [comment]
IMHO like much of litebans notes should take the form of private notes, for example a player says a swear word, it's their first time swearing so they get a verbal warning, it would be nice to be able to do a command and note that they already recieved their first verbal warning via a note that staff can easily check.
